pkgs/ffigen/test/native_objc_test/util.dart

```diff
@@ -39,11 +40,19 @@ final _executeInternalCommand = () {
 bool canDoGC = _executeInternalCommand != null;
 
 void doGC() {
-  final gcNow = "gc-now".toNativeUtf8();
+  final gcNow = 'gc-now'.toNativeUtf8();
   _executeInternalCommand!(gcNow.cast(), nullptr);
   calloc.free(gcNow);
 }
 
+// Dart_ExecuteInternalCommand("gc-now") doesn't work on flutter, so we use
+// leak_tracker's forceGC function instead. It's less reliable, and to combat
+// that we need to wait for quite a long time, which breaks autorelease pools.
+Future<void> flutterDoGC() async {
```

Comment:
Interesting! What does leak tracker do to force GC and why isn't it reliable?

Reply:
Oh! It basically allocates a bunch of object!

Reply:
Yeah. The main issue is that it's async. In some of the tests I'm pushing an autorelease pool, doing a GC, then popping the pool. If the GC is async, then other stuff can run in between pushing and popping the pool. Flutter uses autorelease pools internally, so often they'll end up popping a pool that was pushed before mine, which breaks the stack push/pop pattern the pools are supposed to follow. This causes unexpected ref counts and occasional crashes.